conbench/app/_plots.py

```diff
@@ -209,25 +219,35 @@ def _source(
     points, values_with_unit = [], []
 
     if alert_min:
-        for x in data:
-            alert = 5 * float(x["distribution_stdev"])
-            points.append(float(x["distribution_mean"]) - alert)
+        for x in samples:
+            alert = 5 * float(x.zscorestats.rolling_stddev)
+            points.append(float(x.zscorestats.rolling_mean) - alert)
             values_with_unit.append(unit_fmt(points[-1], unit))
 
     elif alert_max:
-        for x in data:
-            alert = 5 * float(x["distribution_stdev"])
-            points.append(float(x["distribution_mean"]) + alert)
+        for x in samples:
+            alert = 5 * float(x.zscorestats.rolling_stddev)
+            points.append(float(x.zscorestats.rolling_mean) + alert)
```

Comment:
when replacing these I carefully looked at https://github.com/conbench/conbench/blob/aa95b6d52ff6cb6697b7ab8d04c43d6cf3612860/conbench/entities/history.py#L91

I hope I didn't make a mistake here, pls review @austin3dickey 

```python
        return {
            "benchmark_id": history.id,
            "case_id": history.case_id,
            "context_id": history.context_id,
            "mean": _to_float(history.mean),
            "data": data,
            "times": times,
            "unit": history.unit,
            "begins_distribution_change": history.begins_distribution_change,
            "hardware_hash": history.hash,
            "sha": history.sha,
            "repository": history.repository,
            # Note(JP): this is the commit message
            "message": history.message,
            # This is the Commit timestamp. Expose Result timestamp, too?
            "timestamp": history.timestamp.isoformat(),
            "run_name": history.name,
            "distribution_mean": _to_float(history.rolling_mean),
            "distribution_stdev": _to_float(history.rolling_stddev) or 0.0,
        }
```

Reply:
I think you got it right, but the `_to_float()` was doing a little extra work to ensure we returned a float or None (but not NaN). It's probably fine how you have it. `bokeh` will probably figure it out

Reply:
Oh, I got to the code down below where you kept `_to_float()` in the function before returning it from the API. That's great.